dev/lib/edit-map.js

```diff
@@ -109,13 +109,16 @@ export class EditMap {
       events.length = this.map[index][0]
     }
 
-    vecs.push([...events])
+    vecs.push(events.slice())
     events.length = 0
 
     let slice = vecs.pop()
 
     while (slice) {
-      events.push(...slice)
+      // NOTE: do not use spread operator here, it will cause a stack overflow.
```

Comment:
Better than a comment, would be a test that fails with the current logic and passes after

Reply:
I add a large markdown file to `test/fixtures`, it will fail when run the original codes.

Reply:
Thanks, could we remove this comment now, since the test will cover it?